Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -472,8 +544,36 @@ extension _FileManagerImpl {
         }
         return extendedAttrs
     }
-    
+#endif
+
     func attributesOfItem(atPath path: String) throws -> [FileAttributeKey : Any] {
+#if os(Windows)
+        return try path.withNTPathRepresentation { pwszPath in
+            var faAttributes: WIN32_FILE_ATTRIBUTE_DATA = .init()
+            guard GetFileAttributesExW(pwszPath, GetFileExInfoStandard, &faAttributes) else {
+                throw CocoaError.errorWithFilePath(path, win32: GetLastError(), reading: true)
+            }
+
+            let size: UInt64 = (UInt64(faAttributes.nFileSizeHigh) << 32) | UInt64(faAttributes.nFileSizeLow)
+            let creation: Date = Date(timeIntervalSince1970: faAttributes.ftCreationTime.interval)
+            let modification: Date = Date(timeIntervalSince1970: faAttributes.ftLastWriteTime.interval)
+            return [
+                .size: size,
+                .modificationDate: modification,
+                .creationDate: creation,
+                // .posixPermissions: ...,
```

Comment:
Are these a TODO for coming back later (if so, we could leave a TODO comment) or are they just not relevant for Windows (if so, we could leave out the comment and wrap the keys in `#if !os(Windows)`

Reply:
There in lies the question - I would like to believe that we should define them, even on Windows. I would rather that we mark these as TODO - yes, some attributes are unavailable, but, hopefully the combination of unit tests and real world usage would expose those. This allows us to reduce the noise in the build and get to the point where we might be able to draw a line in the sand to avoid backsliding the platform support.

Reply:
Ah gotcha ok yeah I see now. Marking it as TODO looks good then we can always decide which are applicable later once we get it building